.ci/coverity/ci-tasks.sh

```diff
@@ -25,7 +25,7 @@ meson --buildtype=debug \
 pushd coverity
 
 # The coverity scan script returns an error despite succeeding...
- TRAVIS_BRANCH="${TRAVIS_BRANCH:-main}" \
+ TRAVIS_BRANCH="main" \
```

Comment:
Does coverity still require references to travis? Even the script below is named `**travis**ci_build_coverity_scan.sh`.

Reply:
Yes, the script we pull from scan.coverity.com requires TRAVIS_BRANCH.